lib/PuppeteerSharp/ChromeTargetManager.cs

```diff
@@ -264,6 +280,7 @@ private async Task OnAttachedToTarget(object sender, TargetAttachedToTargetRespo
                 _ignoredTargets.Add(targetInfo.TargetId);
                 FinishInitializationIfReady(targetInfo.TargetId);
                 await silentDetach().ConfigureAwait(false);
+                return;
```

Comment:
This seems to fix `PuppeteerConnectTests.ShouldSupportTargetFilter` when running all `PuppeteerConnectTests` in parallel 🚀